### PR TITLE
Add upgrade channel group for a cluster

### PR DESCRIPTION
**Pull request by tzvatot**
@zgalor @igoihman @nimrodshn PTAL

### CODE REVIEW

File: model/clusters_mgmt/v1/cluster_type.model [hunk 1]
```diff
@@ -190,4 +190,7 @@ class Cluster {
 	// Indicates whether that etcd is encrypted or not.
 	// This is set only during cluster creation.
 	EtcdEncryption Boolean
+
+	// Channel group to be used for upgrading the cluster. Valid values: stable (default), fast, candidate, nightly.
+	UpgradeChannelGroup String
```
Comment (zgalor) [inline model/clusters_mgmt/v1/cluster_type.model:195]: Is this patchable?

**Comment by zgalor** [comment]
LGTM

File: model/clusters_mgmt/v1/cluster_type.model (hunk 1 above)
Comment (tzvatot) [reply model/clusters_mgmt/v1/cluster_type.model:195]: will be, yes

**Comment by vkareh** [comment]
LGTM

**Pull request merged**